### PR TITLE
[zh-hans] Add Specifics - Debugging

### PR DESCRIPTION
**Pull request by kenspirit**

### CODE REVIEW

File: zh-hans/lessons/specifics/debugging.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+## Dialyxir 和 Dialyzer
+
+[Dialyzer](http://erlang.org/doc/man/dialyzer.html)，全称就是 **DI**screpancy **A**na**LYZ**er for **ER**lang，它是一个静态代码分析工具。也就是说，它_阅读_和分析你的代码，但是不_执行_它们，比如说，寻找 bugs，无法触及的死代码等。
+
+[Dialyxir](https://github.com/jeremyjh/dialyxir) 则是在 Elixir 里简化了 Dialyzer 使用的 mix 任务。  
+
+Specification 可以帮助像 Dialyzer 这样的工具更好地理解代码。不像那些只能被人类阅读和理解的文档（假如存在或者写的足够好的话），`@spec` 使用了一些可以被机器理解的，更规范的语法。  
+
+让我们把 Dialixyr 加到我们的项目里头吧。最简单的方式就是添加依赖到 `mix.exs` 文件中：
```
Comment (zillou) [inline zh-hans/lessons/specifics/debugging.md]: 这里应该是 `Dialyxir`. 英文原文里面也错了。

Comment (kenspirit) [reply zh-hans/lessons/specifics/debugging.md]: 用的少，只是翻译，还是不够敏感。顺便把  en 和 ru 里面的也改了。谢谢你仔细的 review。

**Comment by doomspork** [comment]
Thank you so much @kenspirit and @zillou 😁

**Comment by zillou** [comment]
@doomspork can we invite @kenspirit to the translation team? He made a lot of contribution to the translation of the Simplified Chinese version.

**Comment by doomspork** [comment]
@zillou indeed!  @elixirschool/chinese, @elixirschool/traditional-chinese, or both?

**Comment by kenspirit** [comment]
@doomspork and @zillou, thanks for your recognition!

**Comment by doomspork** [comment]
@kenspirit you've been added to the Chinese team — welcome!

**Pull request merged**